FreeRTOS/FreeRTOS-Kernel
# Fix ThirdParty/GCC/ATmega formatting

Title
-----------
Fix ThirdParty/GCC/ATmega formatting.

Description
-----------
Unnecessary white space was introduced in #768 which affected the formatting of assembly code.
This PR returns the correct formatting.
No functional change.

Test Steps
-----------
No functional change

Checklist:
----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
